src/briefcase/commands/convert.py

```diff
@@ -326,10 +327,10 @@ def input_url(self, app_name, override_value: str | None) -> str:
 
         return url
 
-    def input_console_app(self, override_value: str | None) -> bool:
+    def input_app_type(self, override_value: str | None) -> bool:
```

Comment:
I was a bit caught off-guard when this didn't work:
```
briefcase convert -Q app_type=console
```
```
-- Interface Style -----------------------------------------------------------

Using override value 'console'

Invalid value; console
Invalid override value 'console' for Interface Style, using user-provided value.

Is this a GUI application or a console application?

  1) GUI
  2) Console

Interface Style [1]:
```
Maybe better to make these comparisons case-insensitive....